alf/config_util.py

```diff
@@ -189,20 +194,8 @@ def is_used(self):
 _CONF_TREE = {}
 
 
-def config1(config_name, value, mutable=True):
-    """Set one configurable value.
-
-    Args:
-        config_name (str): name of the config
-        value (any): value of the config
-        mutable (bool): whether the config can be changed later. If the user
-            tries to change an existing immutable config, the change will be
-            ignored and a warning will be generated. You can always change a
-            mutable config. ``ValueError`` will be raised if trying to set a new
-            immutable value to an existing immutable value.
-
-
-    """
+def _get_config_node(config_name):
+    """Get the _Config corresponding to config_name."""
```

Comment:
Does ```_Config``` has any special meaning here? Maybe can change to ```config node```.

Reply:
_Config is the class name